### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by 0xteet0**
Address Issue #35

Changes
- Add User Guide for marking of student attendance

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -86,6 +89,20 @@ Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
 
+### Marking attendance of student: `mark`
+
+Format: `mark /name STUDENTNAME | /id STUDENTID /attendance ATTENDANCE`
```
Comment (ElginTZM) [inline docs/UserGuide.md:94]: Can mark it as [Coming Soon] as it hasn't been implemented in the current version yet.

**Review by ElginTZM — CHANGES_REQUESTED** [review]
Everything looks good! 👍 Just a small issue with the feature not marked as [Coming Soon].

Edit: Resolved

File: docs/UserGuide.md (hunk 1 above)
Comment (0xteet0) [reply docs/UserGuide.md:94]: Good catch! Will update it now!

**Review by ElginTZM — APPROVED** [review]
LGTM! 👍

**Pull request merged**